packages/components/src/components/Layout.tsx

```diff
@@ -33,11 +34,11 @@ export default function Layout({ children, className, progress, error, title }:
       <div className={cn('content', className)}>
         {children}
       </div>
-      <div className={cn('error-bar', error && 'show-error')}>{errorMessage}</div>
+      <div className={cn('error-bar', error !== undefined && 'show-error')}>{errorMessage}</div>
     </div>
     {progress !== undefined && progress < 1 &&
       <div className={'progress-bar'} role='progressbar'>
-        <div style={{ width: `${100 * progress}%` }} />
+        <div style={{ width: `${(100 * progress).toString()}%` }} />
```

Comment:
Doesn't string interpolation `${...}` implicitly call .toString() anyway? Why the change?

Reply:
it comes from a rule in the strict mode I added in the last commits: https://typescript-eslint.io/rules/restrict-template-expressions/. I also find it annoying. The argument is that it's too implicit. I just disabled that particular rule.